public/app/plugins/datasource/cloudwatch/components/Stats.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { SFC } from 'react';
+import { SelectableValue } from '@grafana/data';
+import { Segment } from '@grafana/ui';
+
+export interface Props {
+  values: string[];
+  onChange: (values: string[]) => void;
+  variableOptionGroup: SelectableValue<string>;
+}
+
+const options: Array<SelectableValue<string>> = ['Average', 'Maximum', 'Minimum', 'Sum', 'SampleCount'].map(value => ({
+  label: value,
+  value: value,
+}));
+
+const removeText = '-- remove stat --';
+const removeOption: SelectableValue<string> = { label: removeText, value: removeText };
+
+export const Stats: SFC<Props> = ({ values, onChange, variableOptionGroup }) => (
```

Comment:
SFC is deprecated (use FunctionalComponent or the short alias FC)

Reply:
👍